libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/generator/OpenHouseDataLayoutStrategyGenerator.java

```diff
@@ -159,6 +173,7 @@ private Optional<DataLayoutStrategy> buildDataLayoutStrategy(
             .eqDeleteFileCount(eqDeleteStats._2())
             .posDeleteRecordCount(posDeleteStats._3())
             .eqDeleteRecordCount(eqDeleteStats._3())
+            .fileCountReductionDiscount(fileCountReductionDiscount)
```

Comment:
s/fileCountReductionDiscount/fileCountReductionPenalty 

Let's use penalty, as we want to penalize partitions that have a high churn rate. Right now, the penalty is set to 0.